package.json

```diff
@@ -7,51 +7,54 @@
   "engines": {
     "node": ">=4.1.0"
   },
-  "devDependencies": {
+  "dependencies": {
     "banner-footer-webpack-plugin": "git://github.com/dtothefp/banner-footer-webpack-plugin.git",
     "browser-sync": "^2.8.0",
-    "cf-component-demo": "git://github.com/cfpb/cf-component-demo.git#0.8.2",
-    "chai": "^3.3.0",
     "dateformat": "^1.0.11",
     "del": "^2.0.0",
     "gulp": "^3.9.0",
     "gulp-autoprefixer": "^3.0.2",
     "gulp-changed": "^1.2.1",
-    "gulp-concat": "^2.6.0",
-    "gulp-coveralls": "^0.1.4",
     "gulp-cssmin": "^0.1.7",
-    "gulp-eslint": "^1.0.0",
     "gulp-header": "^1.2.2",
-    "gulp-imagemin": "^2.3.0",
-    "gulp-istanbul": "^0.10.0",
     "gulp-less": "^3.0.3",
-    "gulp-load-plugins": "^1.0.0-rc.1",
-    "gulp-mocha": "^2.1.3",
+    "gulp-load-plugins": "^1.1.0",
+    "gulp-imagemin": "^2.3.0",
     "gulp-mq-remove": "0.0.2",
     "gulp-notify": "^2.2.0",
-    "gulp-plumber": "^1.0.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-replace": "^0.5.3",
     "gulp-sitespeedio": "^0.0.8",
```

Comment:
This is the error when running `setup.sh production` and having `gulp-sitespeedio` outside of `dependencies`:

```
Building project...
module.js:329
    throw err;
    ^

Error: Cannot find module 'gulp-sitespeedio'
    at Function.Module._resolveFilename (module.js:327:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/Users/bradforda/Projects/cfgov-refresh/gulp/tasks/test.js:5:19)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
```